projects/storefrontlib/src/shared/utils/forms/form-utils.ts

```diff
@@ -40,7 +38,11 @@ export class FormUtils {
    * @param controlName Field name to prefill
    * @param value Value of the filled
    */
-  static prefillForm(form: FormGroup, controlName: string, value: string): void {
+  static prefillForm(
```

Comment:
I wonder about this method. You normally prefill form controls when the form is build.
i.e.

```typescript
this.form = this.fb.group({
      userId: ['HERE IS THE PREFILLED VALUE', [Validators.required, CustomFormValidators.emailValidator]],
      password: ['', Validators.required],
    });
```

If we patch the values afterwards, you'd basically generate a form event, which could cause additional issues. Moreover, it seems to overcomplicate the whole thing.

Reply:
fixed